backend/src/main/java/gov/cdc/usds/simplereport/SimpleReportApplication.java

```diff
@@ -26,6 +28,11 @@
 @EnableScheduling
 public class SimpleReportApplication {
 
+   @PostConstruct
+    public void init(){
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    }
```

Comment:
@benwarfield-usds does this have any strange side effects? This was added because the client was sending over ISO strings (local time + offset), and the server was adding the offset again when writing to the db.